src/rgw/driver/rados/rgw_rados.cc

```diff
@@ -4586,7 +4586,7 @@ int RGWRados::copy_obj_data(RGWObjectCtx& obj_ctx,
   AtomicObjectProcessor processor(&aio, this->driver, &dest_placement,
                                   bucket->get_info().owner, obj_ctx,
                                   dest_obj->clone(), olh_epoch, tag,
-				  dpp, null_yield);
+				  dpp, y);
```

Comment:
@kapandya @cbodley just a heads up I added the comment on the lines above this AtomicObjectProcessor initialization saying that changing this `null_yield` to a `y` causes ragweed tests to fail. I wouldn't be surprised if it happens when we run this PR through teuthology again.